sass/components/_answersWrapper.scss

```diff
@@ -0,0 +1,7 @@
+.answersWrapper {
```

Comment:
Chyba znajdziemy tutaj nazwę bardziej "domenową" jak `questionAnswers` :) ?
Wrapper to takie jak Service / Manager, które można dać wszędzie. Jak można to warto wymyślić coś innego - choć czasem się nie da.

Reply:
> a zmienić to co Piotrek napisał :) Jak myślę - oznaczyć klasą dobra / zła odpowiedź dopiero po kliknięciu

Dokładnie
Jest już zaimplementowane w czekających PR - oopisałem którymś komentaru powyżej